src/seedu/addressbook/data/person/Name.java

```diff
@@ -26,7 +26,8 @@ public Name(String name) throws IllegalValueException {
         if (!isValidName(trimmedName)) {
             throw new IllegalValueException(MESSAGE_NAME_CONSTRAINTS);
         }
-        this.fullName = trimmedName;
+
+        this.fullName = trimmedName.toUpperCase();
```

Comment:
Try not to manipulate data that user has typed. Instead, when implementing case insensitive find, try to use Java's `String .toLowerCase()` method or Java's `Collections.sort()`  with `String.CASE_INSENSITIVE_ORDER`, which will not directly modify stored data (: